### PR TITLE
:memo: add markdown on the ctf folder

### PR DESCRIPTION
**Pull request by stypr**
Yes here you go :)
(From: https://github.com/ctfs/write-ups-2015/pull/93)

### CODE REVIEW

**Comment by YASME-Tim** [comment]
Thanks :)

File: codegate-ctf-2015/README.md [hunk 1]
```diff
@@ -6,7 +6,8 @@
 
 ## Completed write-ups
 
-* none yet
+* [web/owlur](web/owlur)
+* [web/owltube](web/owltube)
 
 ## External write-ups only
```
Comment (dhanvi) [inline codegate-ctf-2015/README.md]: you forgot to delete them from the external write-ups only :-1:

**Comment by stypr** [comment]
oops, i will do that by tomorrow. its 5am here maybe I was in a lack of sleep :p

**Comment by YASME-Tim** [comment]
Done :)

**Pull request merged**